tools/cli/go-whisk/whisk/client.go

```diff
@@ -307,10 +307,10 @@ func parseWhiskErrorResponse(resp *http.Response, data []byte, v interface{}) (*
     err := json.Unmarshal(data, whiskErrorResponse)
 
     // Determine if a whisk.error() response was received. Otherwise, the body contents are unknown (#6)
-    if err == nil && whiskErrorResponse.Response.Result.Error != nil {
-        Debug(DbgInfo, "Detected that a whisk.error(\"%s\") was returned\n", *whiskErrorResponse.Response.Result.Error)
+    if err == nil && whiskErrorResponse.Response.Status != nil {
```

Comment:
can you confirm that this will only run when the HTTP status code for the invoke is not 200?